godot-kotlin/godot-library/src/nativeCore/kotlin/godot/core/bridge.kt

```diff
@@ -0,0 +1,54 @@
+package godot.core
+
+import godot.gdnative.godot_variant
+import kotlinx.cinterop.*
+
+fun createInstance(instance: COpaquePointer?, methodData: COpaquePointer?): COpaquePointer? {
+    val classHandle = checkNotNull(methodData).asStableRef<ClassHandle<Object>>()
+        .get()
+    val kotlinInstance = classHandle.wrap(checkNotNull(instance))
+    kotlinInstance._onInit()
+    val stableRef = StableRef.create(kotlinInstance)
+    return stableRef.asCPointer()
+}
+
+fun disposeClassHandle(ref: COpaquePointer?) {
+    val handle = checkNotNull(ref).asStableRef<ClassHandle<Object>>()
+    handle.get().dispose()
+    handle.dispose()
+}
+
+fun destroyInstance(instance: COpaquePointer?, methodData: COpaquePointer?, classData: COpaquePointer?) {
+    val kotlinInstanceRef = checkNotNull(classData).asStableRef<Object>()
+    val kotlinInstance = kotlinInstanceRef.get()
+    kotlinInstance._onDestroy()
+    kotlinInstanceRef.dispose()
+}
+
+fun invokeMethod(instance: COpaquePointer?,
+                 methodData: COpaquePointer?,
+                 classData: COpaquePointer?,
+                 numArgs: Int,
+                 args: CPointer<CPointerVar<godot_variant>>?): CValue<godot_variant> {
```

Comment:
In kotlin styleguide, aren't all params on a separate line?
(
param,
param,
)

Reply:
Yeah, just copied it from my binding.

Reply:
I don't think it is